tests/reference/asr-string_02-499c9ff.stdout

```diff
@@ -0,0 +1 @@
+(Var 3 p)
```

Comment:
The CI tests seem to pass. I think we just need to remove this `stdout` output. For the `string_02` error test, as it results in an error, we would/should not have any `stdout` file generated for it.

You can simply remove this specific file and commit the changes.

Reply:
okay, sure